harvester_e2e_tests/integration/test_images.py

```diff
@@ -0,0 +1,212 @@
+
+from harvester_e2e_tests.apis import test_images, test_volumes
+
+from time import sleep
+
+import pytest
+import os
```

Comment:
https://peps.python.org/pep-0008/#imports

Reply:
Thanks for the review. 
Remove `import os` and corresponding function `upload_file`